src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormArtistStep.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Spacer, Text } from "@artsy/palette"
+import { AppContainer } from "Apps/Components/AppContainer"
+import { useMyCollectionArtworkFormContext } from "Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormContext"
+import { MyCollectionArtworkFormHeader } from "Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormHeader"
+
+interface MyCollectionArtworkFormArtistStepProps {}
+
+export const MyCollectionArtworkFormArtistStep: React.FC<MyCollectionArtworkFormArtistStepProps> = () => {
+  const { onBack, onNext, onSkip } = useMyCollectionArtworkFormContext()
+
+  return (
+    <>
```

Comment:
**suggestion:** maybe we can remove this 
```suggestion
```

Reply:
Good idea! I've removed the `<></>`